meterfs/file_pc.c

```diff
@@ -0,0 +1,141 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs test file abstraction
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ * %LICENSE%
+ */
+
+#include <string.h>
+#include <limits.h>
+#include <host-flashsrv.h>
+#include <meterfs.h>
+
+#include "file.h"
+
+#define FLASHSIZE 4*1024*1024
```

Comment:
don't spotted it before
(4 * 1024 * 1024)

it's much safer to have define in ()

Reply:
Done.